lib/controllers/game_controller.cpp

```diff
@@ -46,12 +53,12 @@ void GameController::createSystems() {
 }
 
 void GameController::createTestEntities() {
-    entityFactory->createPanda(400, 200, 1);
-    entityFactory->createGorilla(1000, 200, 2);
-    entityFactory->createImage("backgrounds/forest_watermarked.jpg", SCREEN_WIDTH / 2, SCREEN_HEIGHT / 2, SCREEN_WIDTH, SCREEN_HEIGHT, Layers::Background);
-    entityFactory->createPlatform(1200, 680, 400, 10);
-    entityFactory->createPlatform(400, 680, 400, 10);
-    entityFactory->createPlatform(800, 510, 800, 10);
+    entityFactory->createPanda(0, 0, 1);
+    entityFactory->createGorilla(0, 0, 2);
+
+    Json level_json = Json("assets/levels/level2.json", true);
+    auto level = std::unique_ptr<Level>(new Level(level_json, SCREEN_WIDTH, SCREEN_HEIGHT));
```

Comment:
No need to allocate this on heap, just make it a local variable and then pass it to the loadLevel function.